xla/tests/collective_ops_test_e2e.cc

```diff
@@ -24,6 +24,7 @@ limitations under the License.
 #include "xla/hlo/utils/hlo_matchers.h"
```

Comment:
Could you maybe rename this test file to collective_ops_e2e_test.cc? Our infra is setup to check for test.cc suffix for verifying that a change contains also a test change.

Reply:
I have just changed that in https://github.com/openxla/xla/commit/812d0f931cd201ead20c86d717e5555ea2e96250. 
@wenscarl Could you please help rebase?

Reply:
Rebase is done.